cloud/azure/variables.tf

```
@@ -44,6 +44,12 @@ variable "excluded_services" {
   default     = []
 }
 
+variable "additional_services " {
+  description = "Azure service metrics to import as custom metrics for services not available in SFX"
```

the variables does not take metrics as input but resource types
```suggestion
  description = "Not yet officially supported azure resource types to sync with SignalFx as custom metrics"
```